readthedocs/templates/profiles/private/token_list.html

```diff
@@ -9,7 +9,7 @@
 {% block edit_content_header %} {% trans "API Tokens" %} {% endblock %}
 
 {% block edit_content %}
-  <p>Personal Access Token are tokens that allow you to use the Read the Docs APIv3 being authenticated as yourself. See <a href="https://docs.readthedocs.org/page/api/v3.html">APIv3 documentation</a> for more information.</p>
+  <p>{% trans "Personal Access Token are tokens that allow you to use the Read the Docs APIv3 being authenticated as yourself. See <a href="https://docs.readthedocs.org/page/api/v3.html">APIv3 documentation</a> for more information." %}</p>
```

Comment:
You would have to put this in a `{% blocktrans %}` instead, it raises a syntax errors since the string contains `"`.

Reply:
Done, see 9eede50